### PR TITLE
feat: Grants sheet modification

### PR DESCRIPTION
**Pull request by ncomerci**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -62,6 +62,12 @@ export function dayToMillisec(dayAmount: number) {
   return dayAmount * 24 * 60 * 60 * 1000
 }
 
+export function getMonthsBetweenDates(startDate: Date, endDate: Date) {
```
Comment (1emu) [inline src/utils.ts:65]: Let's add some tests for this fn please :) there might me some interesting edge cases

File: src/interfaces/Grant.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export interface VestingInfo {
   vesting_token_contract_balance: number
   vesting_total_amount: number
   vesting_status: VestingStatus
+  duration: number
```
Comment (1emu) [inline src/interfaces/Grant.ts:26]: I'm thinking this name should explicit the units of the duration somehow. 

How about `months_duration` or `duration_in_months`?

**Review by 1emu — CHANGES_REQUESTED** [review]

**Review by andyesp — APPROVED** [review]

**Review by 1emu — APPROVED** [review]
👏  love the tests <3
